interpreter/cling/lib/Interpreter/DynamicLibraryManager.cpp

```diff
@@ -142,8 +159,26 @@ namespace cling {
     return NPath;
   }
 
-  std::string
-  DynamicLibraryManager::lookupLibrary(llvm::StringRef libStem) const {
+  ///\returns true if substitution was successful.
+  static bool subst(llvm::StringRef value, std::string find,
+                    llvm::StringRef substWith, std::string& result) {
+    if (value.startswith_lower(find)) {
```

Comment:
If you decide to return `string`, consider inverting:
```suggestion
    if (!value.startswith_lower(find))
      return find;
```